meta/autoid/autoid.go

```diff
@@ -890,3 +892,41 @@ func EncodeIntToCmpUint(v int64) uint64 {
 func DecodeCmpUintToInt(u uint64) int64 {
 	return int64(u ^ signMask)
 }
+
+// AutoRandomIDLayout is used to calculate the bits length of different section in auto_random id.
+// Layout(64 bits):
+// [zero_padding] [sign_bit] [shard_bits] [incremental_bits]
```

Comment:
Do we have lead-padding zero in auto_random now?

Reply:
Yep. If the column type is not `bigint`, there is zero-padding in 64-bits layout.